airflow/api_connexion/schemas/dag_run_schema.py

```diff
@@ -72,5 +72,25 @@ class DAGRunCollectionSchema(Schema):
     total_entries = fields.Int()
 
 
+class DagRunsBatchFormSchema(Schema):
+    """ Schema to validate and deserialize the Form(request payload) submitted to DagRun Batch endpoint"""
+
+    class Meta:
+        """ Meta """
+        datetimeformat = 'iso'
+        strict = True
+
+    page_offset = fields.Int(required=False, missing=0, min=0)
+    page_limit = fields.Int(required=False, missing=100, min=1)
```

Comment:
```suggestion
    page_limit = fields.Int(required=False, missing=100, min=1, max=100)
```

It would be better to cap the limit to 100. @ephraimbuddy @takunnithan what do you think?

Reply:
I think it's better left out. Some servers can handle more than 1000 page request limit. I'm working on configurable page request limit and we can apply it here. Then it'll not be a hard limit but a limit that the users can configure.
You can see the commit here https://github.com/apache/airflow/commit/a37ac22c179e5dccc95b05c862bffc542dad125e